common/api/summary/appui-abstract.exports.csv

```diff
@@ -3,38 +3,53 @@ Release Tag;API Item
 public;AbstractActionItemProps 
 public;AbstractMenuItemProps 
 public;AbstractStatusBarActionItem 
+deprecated;AbstractStatusBarActionItem 
 public;AbstractStatusBarCustomItem 
+deprecated;AbstractStatusBarCustomItem 
 public;AbstractStatusBarItem 
+deprecated;AbstractStatusBarItem 
 public;AbstractStatusBarItemUtilities
+deprecated;AbstractStatusBarItemUtilities
 public;AbstractStatusBarLabelItem 
+deprecated;AbstractStatusBarLabelItem 
 public;AbstractToolbarProps
```

Comment:
Did not deprecate `AbstractToolbarProps` due to `UiAdmin.showToolbar`. Toolbars are also defined in `components-react`.